editors/code/src/installation/download_file.ts

```diff
@@ -1,6 +1,10 @@
 import fetch from "node-fetch";
 import * as fs from "fs";
 import { strict as assert } from "assert";
+import { NestedError } from "ts-nested-error";
+
+
```

Comment:
I'd remove the extra line of white space.

Reply:
Will do!